charts/neuvector/latest/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: neuvector
+apiVersion: v1
+version: 1.2.3
+appVersion: 2.4.2
+description: NeuVector Container Security Platform includes layer 7 container firewall, end-to-end vulnerability scanning, and container process/file monitoring.
+home: https://neuvector.com
+maintainers:
+  - name: Sun
+    email: xfsun@neuvector.com
+engine: gotpl
```

Comment:
A logo is always preferred, but not required in here.

Reply:
Logo will be added in the updated PR.